scalafix/rules/src/main/scala/rsc/rules/pretty/SemanticdbPrinter.scala

```diff
@@ -12,10 +12,25 @@ import scala.meta.internal.{semanticdb => s}
 import scala.meta.internal.semanticdb.Scala._
 import scala.meta.internal.semanticdb.Scala.{Descriptor => d}
 import scala.meta.internal.semanticdb.Scala.{Names => n}
+import scala.meta.internal.semanticdb.{SymbolInformation, TypeRef, TypeSignature}
 import scalafix.internal.v0._
 
 class SemanticdbPrinter(env: Env, index: DocumentIndex, config: RscCompatConfig) extends Printer {
   def pprint(tpe: s.Type): Unit = {
+    def equivalent(name: Option[n.Name], sym: String): Boolean = {
+      name
+        .flatMap(env.lookupInfo)
+        .exists { info =>
+          info match {
+            case SymbolInformation(_, _, kind, _, _, TypeSignature(_, TypeRef(_, lbsym, _), TypeRef(_, ubsym, _)), _, _) =>
+              lbsym == ubsym && lbsym == sym
+
+            case _ =>
+              info.symbol == sym
+          }
+        }
+    }
```

Comment:
I'd move this logic to an extension method on `Symtab` in `Symtabs.scala`. It seems to be completely isolated from the logic of rewrite, so I think it'd be good to move it away.